src/components/Tag/index.ts

```diff
@@ -1 +1,2 @@
 export {Tag} from './Tag.component';
+export type {TagProps, LabelTag, HighlightTag, ShortcutTag} from './Tag';
```

Comment:
same

Reply:
Exported all because `TagProps' can be either `LabelTag`, `ShortcutTag`, or `HighlightTag`. If we want `TagProps ` to be only `LabelTag` type on the plugin side then we have to specify type there.